src/gtk/textctrl.cpp

```diff
@@ -1525,6 +1559,12 @@ void wxTextCtrl::GTKOnTextChanged()
 {
     if ( IgnoreTextUpdate() )
         return;
+    if( IsMultiLine() )
+    {
+        auto count = gtk_text_buffer_get_char_count( m_buffer );
+        if( m_maxlen > 0 && count >= m_maxlen )
```

Comment:
Why is `>=` used here? Shouldn't it be `>`?

```suggestion
        if ( m_maxlen > 0 && count > m_maxlen )
```

Reply:
@vadz , Changing this will bring back bad event generation behavior.